QUEST.tsv

```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
-QUEST_20150317_001170	People who model Stars Mr. encounters that were walking down the street and pull to say so {nl} was the soldiers of the people number. Soldiers to {nl} model Stars san see looked to the hassle to put something. {nl} model Stars Mr., was Jinagaryo only whether they are not, soldiers cried when you see the model's Stars. {nl}
-QUEST_20150317_001171	There will pull the words to Uncle. Come, do not me help us. {nl} model Stars Mr., the number of the soldiers did not know what helped me in any thing, the sound to hear it went closer to them to collect the words, rather than be able to pass as it is. {nl} Once the model Stars Mr. approaches, one of the soldiers said. {nl} "just good come small. Cattle Once you have me look at the referee that we were one of the bet here." {nl}
+QUEST_20150317_001158	Anyhow, as Modestas and the Old Lady parted ways, he knew he now had a dangerous power guaranteeing his victory at any time in the near future. {nl} He walked, looking for the next person to bet huge on. As he wondered how much to bet on his next move, he heard the sound of running footsteps behind him.
+QUEST_20150317_001159	Modestas looked back, it was his good old friend Sigfried on horseback riding towards him. {nl} Sigfried and Modestas knew each other, as both had been together through the highs and lows of gambling. They greeted each other, and talked about where they were going. To walk with his good friend Modestas, Sigfried got off his horse. They walked at a leisurely pace, with idle chatter along the way. {nl}
+QUEST_20150317_001160	However, before long, Modestas could no longer continue walking with Sigfried. He was fully aware of his deadline, and had no time for trivial activities with Sigfried. {nl}
+QUEST_20150317_001161	Even during their normal everyday talk, Modestas and Sigfried unconsciously made big and small bets. Eventually, one would utter the word, "bet?" without knowing he did. {nl} Both Sigfried and Modestas weren't rich, therefore, there was no way this bet could be big. Knowing this could trigger the curse, Modestas really wanted to simply tell Sigfried he's gotta go. {nl} In the end, Modestas uttered these words. {nl}
+QUEST_20150317_001162	"Are you not busy?" You seemed busy on your horse before you passed by for a talk. You should stop worrying about me and go." {nl} Sigfried, hearing Modestas's words, replied, {nl} "I was busy, but I still have time to walk with you for another 10 minutes or so. This horse might look look average, but when it comes to it he can go pretty fast." {nl}
```

Comment:
"Are you not busy?" -> "Are you not busy?
There's an extra quotation mark there.

look look -> look